README.md

```diff
@@ -126,17 +145,29 @@ buildFailed: Boolean
 
 ## Build from source
 
-### prerequisites
+### Prerequisites
 
 * install java 8+
 * install scala 2.11+
 * install SBT ([www.scala-sbt.org](http://www.scala-sbt.org))
 
-### run from sbt
+### Run from sbt
 
 `sbt "run-main io.magentys.donut.Boot -s cucumber:/my/path/cucumber-reports -n myProjectName" `
 
-### credits
+### Build Docker Container
+
+Set your Docker Hub credentials
+```
+export VERSION=version_number

```

Comment:
These are manual commands at the moment, and need to tally the version.

Next steps will be to automatically build, and will need to get the version from SBT